commands/prune.go

```diff
@@ -149,12 +150,13 @@ func pruneCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&options.all, "all", "a", false, "Include internal/frontend images")
 	flags.Var(&options.filter, "filter", `Provide filter values (e.g., "until=24h")`)
-	flags.Var(&options.minStorage, "min-storage", "Minimum amount of disk space to keep for cache")
-	flags.Var(&options.maxStorage, "max-storage", "Maximum amount of disk space to keep for cache")
+	flags.Var(&options.reservedSpace, "reserved-space", "Amount of disk space always allowed to keep for cache")
+	flags.Var(&options.minFreeSpace, "min-free-space", "Target amount of free disk space after pruning")
+	flags.Var(&options.maxUsedSpace, "max-used-space", "Maximum amount of disk space allowed to keep for cache")
 	flags.BoolVar(&options.verbose, "verbose", false, "Provide a more verbose output")
 	flags.BoolVarP(&options.force, "force", "f", false, "Do not prompt for confirmation")
 
-	flags.Var(&options.maxStorage, "keep-storage", "Amount of disk space to keep for cache")
+	flags.Var(&options.reservedSpace, "keep-storage", "Amount of disk space to keep for cache")
```

Comment:
We can mark this hidden I think.

There is no problem with both using `&options.reservedSpace` as target? Maybe we should check this separately and also detect if both flags are set.

Reply:
`MarkDeprecated` makes it hidden too along with a message. I don't believe there's any problem with both being set. The second will just overwrite the first and then they'll get a deprecated message. I don't think we need to detect if both are set because they should have the same behavior.